go.mod

```diff
@@ -1,10 +1,9 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib
 
-go 1.22.0
-
-toolchain go1.22.2
+go 1.22.4
```

Comment:
There are some inconsistencies in the go.mods. Not sure why this one removed the `toolchain`. The `confighttp`/`configgrpc` use 1.21 too.